src/goose/api/batch.clj

```diff
@@ -0,0 +1,27 @@
+(ns goose.api.batch
+  (:require [goose.broker :as b]
+            [goose.client]))
+
+(defn- status-from-counts
+  [{:keys [successful dead]}
+   total]
+  (if (= (+ successful dead) total)
+    "complete"
```

Comment:
suggestion: We can define the status as `def` in batch file to be more explicit about the possible state of a batch.